<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSetPlayerUnitsMenu</class>
 <widget class="QDialog" name="CSetPlayerUnitsMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>735</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="headline">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>421</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Spielereinheiten setzen</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="CMyView" name="countryView">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>531</width>
     <height>391</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="amountToSetLabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>550</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Anzahl der Einheiten zu setzen:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="unitsSB">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>550</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="setPB">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>620</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Setzen!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quitPB">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>620</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Abbrechen</string>
   </property>
  </widget>
  <widget class="QLabel" name="unitsAvailableLable">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>510</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Verf√ºgbare Einheiten:</string>
   </property>
  </widget>
  <widget class="QLabel" name="unitsAvailableAmountLable">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>510</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="indent">
    <number>0</number>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CMyView</class>
   <extends>QGraphicsView</extends>
   <header>myview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
